acquia_cms.profile

```diff
@@ -382,9 +302,8 @@ function acquia_cms_rebuild_site_studio() {
 function acquia_cms_rebuild_site_studio_finished() {
   // The 'success' parameter means no fatal PHP errors were detected. All
   // other error management should be handled using 'results'.
```

Comment:
@vishalkhode1 Can we alter the comments here a bit? Are these comments related to Telemtery class itself?

Reply:
No, that's a general comment on how batch API works. That is, it returns the results array. If array contains key: `error` means, batch doesn't executed successfully and it reported some error.

Reply:
But yes, ideally it shouldn't be there.